CHANGELOG.md

```diff
@@ -0,0 +1,175 @@
+<a name="4.0.1"></a>
+## [4.0.1](https://github.com/osquery/osquery/releases/tag/4.0.1)
+
+This release has two major focuses.  It is first release since [osquery transitioned to being part of the linux foundation](https://www.linuxfoundation.org/press-release/2019/06/the-linux-foundation-announces-intent-to-form-new-foundation-to-support-osquery-community/).
+
+It features a heavily reworked build system. This aims to provide flexibility and stability. 
+
+[Git Commits](https://github.com/osquery/osquery/compare/3.3.2...4.0.1)
+
+
+### New Features / Under the Hood improvements
+
+Audit — `process_events` Implement support for fork/vfork/clone/execveat ([#5701](https://github.com/osquery/osquery/pull/5701))
+
+New SQLite function `regex_match` to match across columns ([#5444](https://github.com/osquery/osquery/pull/5444))
+
+macOS query pack: detect SearchAwesome malware ([#5713](https://github.com/osquery/osquery/pull/5713))
+
+macOS query pack: detect when a process is tapping keyboard event ([#5345](https://github.com/osquery/osquery/pull/5345))
+
+LRU cache for syscall tracing ([#5521](https://github.com/osquery/osquery/pull/5521))
+
+Basic tracing via eBPF on Linux ([#5403](https://github.com/osquery/osquery/pull/5403), [#5386](https://github.com/osquery/osquery/pull/5386), [#5384](https://github.com/osquery/osquery/pull/5384)) 
+
+Experimental `kill` and `setuid` syscall tracing in Linux via eBPF ([#5519](https://github.com/osquery/osquery/pull/5519))
+
+New eventing (ev2) framework ([#5401](https://github.com/osquery/osquery/pull/5401))
+
+Increase the amount of MaxRecvRetries for thrift socket ([#5390](https://github.com/osquery/osquery/pull/5390))
+
+Added CodeProfiler
+
+
+### Build
+
+Add CMake support ([#5604](https://github.com/osquery/osquery/pull/5604), [#5627](https://github.com/osquery/osquery/pull/5627), [#5630](https://github.com/osquery/osquery/pull/5630))
```

Comment:
I would list this a refactor maybe, since CMake was actually never lost if we compare the latest stable releases (3.3.2/3.3.3 -> 4.0.0/4.0.1)